package.json

```diff
@@ -45,8 +45,8 @@
         "release": "npm run build && lerna publish from-package --contents dist",
         "publish:next": "lerna publish --canary --preid beta --dist-tag next",
         "release:next": "npm run build && npm run publish:next",
-        "lint": "eslint packages/*/src test",
-        "lint:fix": "eslint packages/*/src test --fix"
+        "lint": "eslint packages/*/src packages/*/*/src test",
+        "lint:fix": "eslint packages/*/src packages/*/*/src test --fix"
```

Comment:
FYI the globs should be wrapped in quotes, otherwise it gets expanded by shell which can have quite significant perf cost

https://github.com/apify/crawlee/blob/master/package.json#L49-L50

https://eslint.org/docs/latest/use/command-line-interface#run-the-cli

Reply:
Ah, nice, eslint can process the globs itself, that simplifies it 👍️